geanylatex/src/geanylatex.c

```diff
@@ -2128,8 +2128,8 @@ add_menu_to_menubar(void)
 	{
 		/* Build up menu for menubar */
 		menu_latex = gtk_menu_item_new_with_mnemonic(_("_LaTeX"));
-		gtk_menu_shell_insert(
-			menubar, menu_latex, g_list_length(menubar->children)-1);
+		gtk_menu_shell_append(
+			menubar, menu_latex);
```

Comment:
Is this the same?  I think that was the same change that altered GeanyVC behavior, wasn't it?